### PR TITLE
feat: add CORS flag

### PR DESCRIPTION
**Pull request by eddycharly**
This PR adds a flag to enable/disable CORS.

### CODE REVIEW

**Review by fjogeleit — DISMISSED** [review]

File: backend/pkg/server/server.go [hunk 1]
```diff
@@ -33,18 +33,20 @@ type server struct {
...
-	router.Use(cors.New(cors.Config{
-		AllowOrigins:  []string{"*"},
-		AllowMethods:  []string{"POST"},
-		AllowHeaders:  []string{"Origin", "Content-Type"},
-		ExposeHeaders: []string{"Content-Length"},
-	}))
+	if enableCors {
+		router.Use(cors.New(cors.Config{
+			AllowOrigins:  []string{"*"},
+			AllowMethods:  []string{"POST"},
```
Comment (fjogeleit) [inline backend/pkg/server/server.go:45]: we should extend this to POST and GET

Comment (eddycharly) [reply backend/pkg/server/server.go:45]: what do you mean ?
does CORS applies to GET ?

Comment (eddycharly) [reply backend/pkg/server/server.go:45]: done

Comment (fjogeleit) [reply backend/pkg/server/server.go:45]: yes, I think so.

Comment (eddycharly) [reply backend/pkg/server/server.go]: added `HEAD` too

**Review by fjogeleit — APPROVED** [review]

**Review by fjogeleit — APPROVED** [review]

**Pull request merged**